BotBrownCore/Configuration/UsernameConfiguration.cs

```diff
@@ -4,15 +4,15 @@
     using System.Collections.Generic;
     using System.ComponentModel;
 
-    public sealed class UsernameConfiguration : IConfiguration
+    public sealed class UsernameConfiguration : IChangeableConfiguration
     {
-        public Dictionary<string, ChannelUser> Users { get; set; } = new Dictionary<string, ChannelUser>();
```

Comment:
Same for this dictionary. In the SimpleTextCommand you added an init method. Should we add this here, too? But isn't the initialization right at here the better option as this property is accessible from outside?